signer/src/transaction_coordinator.rs

```diff
@@ -248,6 +255,11 @@ where
             None => self.coordinate_dkg(&bitcoin_chain_tip).await?,
         };
 
+        if with_delay > tokio::time::Duration::ZERO {
```

Comment:
bug? I think this needs to be moved before the above block coordinating DKG since it depends on the Bitcoin chain tip (which is what we're worried that the other signers may not have updated yet hence the delay).

Reply:
Uh, I think you are right. Done.